s3commander.js

```diff
@@ -641,8 +641,14 @@ b64pad = "=";
       // file control
       return (
         <div className={this.props.style.entry}>
-          <span className="glyphicon glyphicon-file"></span>
-          <a onClick={this.onDownload}>{file.name}</a>
+          <span
+            onClick={this.onDownload}
+            className="s3icon glyphicon glyphicon-file">
```

Comment:
s3icon class makes the cursor a pointer when mousing over the file icon. This doesn't need to be on link because btn-link class already does this.